src/object_classes_permissions.md

```diff
@@ -2143,6 +2150,26 @@ Manage the Binder IPC service.
 - Transfer a binder reference to another process (can A transfer a binder
   reference to B?).
 
+### *io_uring*
+
+Manage security-sensitive usages of the *io_uring* subsystem.
+
+**Permission** - 3 unique permissions:
+
+*override_creds*
+
+- Use another process's credentials via an *io_uring* personality (Can A use B's
+  credentials when calling ***io_uring_enter**(2)*?).
+  Personalities are explained in ***io_uring_register**(2)*.
```

Comment:
In addition to pointing readers at `io_uring_register(2)`, it might also be nice to provide a quick explanation that one can think of io_uring personalities as credential references which allow one to store a copy of their credentials, including SELinux label, in an io_uring for use by other processes.

It still scares me every time I think of that bit of functionality ...